packages/lesmis/src/views/DashboardView.js

```diff
@@ -21,7 +21,7 @@ import {
   PartnerLogo,
 } from '../components';
 import { useUrlParams, useUrlSearchParams } from '../utils';
-import { useVitalsData, useEntityData } from '../api/queries';
+import { useVitalsData, useEntityData, useUser } from '../api/queries';
```

Comment:
Is this new import being used in a later PR maybe?

Reply:
nope so I've removed it